ktor-io/common/src/io/ktor/utils/io/ByteReadChannelOperations.kt

```diff
@@ -442,20 +442,33 @@ public suspend inline fun ByteReadChannel.read(crossinline block: suspend (ByteA
 public val ByteReadChannel.availableForRead: Int
     get() = readBuffer.buffer.size.toInt()
 
+
 /**
- * Reads all [length] bytes to [dst] buffer or fails if channel has been closed.
- * Suspends if not enough bytes available.
+ * Reads bytes available bytes into the provided [out] buffer, or fails
```

Comment:
Nit: typo
```suggestion
 * Reads all available bytes into the provided [out] buffer, or fails
```

Reply:
I just dropped the `readFully` with sans args in favor of default args for simplicity.